pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -734,6 +734,8 @@ func (c *Controller) AppendInstanceHandler(f func(*model.ServiceInstance, model.
 			return nil
 		}
 		c.updateEDS(ep)
+
+		log.Infof("Handle endpoint %s in namespace %s", ep.Name, ep.Namespace)
```

Comment:
? this is new? And this was removed because pilot console was flooded. Atleast make it debug or something.

Reply:
It is not new - it was there, it was removed, I'm adding it back
It's very important to know when endpoint changes are visible - saved a lot of guessing